lib/elixir_search_extractor_web/templates/keyword_file/index.html.eex

```diff
@@ -1,28 +1,32 @@
-<h1>Listing Keyword files</h1>
+<div class="page-header">
```

Comment:
Let's fix the spacing in this file. It looks a bit wider.

Reply:
Also, fix in other files too.

Reply:
Fixed issues in [54c58ea](https://github.com/rafayet-monon/elixir_search_extractor/pull/36/commits/54c58ea81b3b8862d92e925946d9dc725a1c90ff)